models/articles-model.js

```diff
@@ -91,3 +91,16 @@ exports.updateArticleByID = (article_id, inc_votes) => {
       return updatedArticleData;
     });
 };
+
+exports.fetchCommentsByID = (article_id) => {
+  return db
+  .query(`SELECT comments.* FROM comments LEFT JOIN articles ON comments.article_id = articles.article_id WHERE comments.article_id = $1 ORDER BY comments.created_at DESC;`, [article_id])
```

Comment:
we don't necessarily need a join here as all the information we need is on the comments so here we can just `select from comments where review_id = 4`  and then keep the order by!